Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -615,12 +615,14 @@ public struct ObjcLibrary: BazelTarget, UserConfigurable, SourceExcludable {
             )
 
         // Include headders
-        var iquotes = [String]()
-        for searchPath in headerSearchPaths {
+        let iquotes = headerSearchPaths
+            .sorted { $0 < $1 }
```

Comment:
this seems to be a pretty recycled closure, what about making an extension onto collection?

Reply:
In many cases, we're sorting on custom types like `AttrSet` etc, so it'd need to be added to many places. Perhaps, we can compose `.sorted` with `<` to clean it up a bit.